lib/pump.c

```diff
@@ -0,0 +1,132 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+//Read Data and convert into Struct
+const int NumSensors = 5;
+
+const int A =20;
+const int B =15;
+const int C =10;
+const int D =5;
```

Comment:
what do these mean? (plus, remember in c everything is global, so you should better use something like PUMP_THRESHOLD_VERYLOW, _LOW or whatever)

Reply:
the numers are picked randomly just to test the code, we should change them for the real values that we consider "border levels" of the sensors